natalie.swift

```diff
@@ -773,11 +773,11 @@ class StoryboardFile {
     }
 
     func processStoryboard() {
-        if let xml = self.xml, viewControllers = searchAll(xml, "sceneMemberID", "viewController") {
-            for viewController in viewControllers {
+        if let xml = self.xml, viewControllers = searchAll(xml, "sceneMemberID", "viewController"), scenes = searchAll(xml, "sceneID") {
```

Comment:
is this `viewControllers = searchAll(xml, "sceneMemberID", "viewController")` still necessary ?